test/integration/targets/postgresql/tasks/main.yml

```
@@ -724,6 +724,49 @@
       - "result_database.stdout_lines[-1] == '(1 row)'"
       - "'{{ db_user2 }}=Cc' in result_database.stdout"
 
+- name: Create function for test
+  become: True
+  become_user: "{{ pg_user }}"
+  postgresql_query:
+    login_user: "{{ db_user1 }}"
+    login_password: password
+    db: "{{ db_name }}"
+    query: CREATE FUNCTION public.a() RETURNS integer LANGUAGE SQL AS 'SELECT 2';
+
+- name: Grant execute to all functions
+  postgresql_privs:
+    type: "function"
```

Remove quotes from where they are not necessary like, etc
```suggestion
    type: function
```

---

I was doing what is done in the rest of the tests, I agree no `"` looks better.